crypto/dilithium/p_pqdsa_test.cc

```diff
@@ -403,8 +1023,14 @@ TEST_P(PQDSAParameterTest, KAT) {
 
     // Generate key pair from seed xi and assert that public and private keys
     // are equal to expected values from KAT
-    if (name == "MLDSA65") {
-      ASSERT_TRUE(ml_dsa_65_keypair_internal(pub.data(), priv.data(), xi.data()));
+    if (name == "MLDSA44") {
+      ASSERT_TRUE(ml_dsa_44_keypair_internal(pub.data(),priv.data(),xi.data()));
```

Comment:
nit: missing spaces after the commas.

Reply:
fixed in [cd9467c](https://github.com/aws/aws-lc/pull/2009/commits/cd9467c75ae0a821173c3feace012a8f28e507ba) (redo of this part of the code)